src/engine/machines/analyzer.rs

```diff
@@ -46,7 +46,7 @@ impl DependencyAnalyzer {
                 let vulnerabilities: Vec<_> = db
                     .query(&query)
                     .into_iter()
-                    .filter(|vuln| !vuln.metadata.yanked)
+                    .filter(|vuln| vuln.metadata.withdrawn.is_none())
```

Comment:
Nitpick: this convenience method should make it easier to read

```suggestion
                    .filter(|vuln| !vuln.withdrawn())
```